<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_Refresh">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Reload Current Folder FBXs</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="led_Folder">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_BrowseFolder">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Choose Working Folder</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_OpenFolder">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Open Working Folder in Explorer</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="lst_Files">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Import">
       <property name="toolTip">
        <string>Import Selected FBXs</string>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Options">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Options</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="wdg_OptionsContent" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chk_CleanAttributes">
            <property name="toolTip">
             <string>Clean Imported User Defined and FBX Attributes</string>
            </property>
            <property name="text">
             <string>Clean Attributes</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chk_FlipAxis">
            <property name="toolTip">
             <string>Flip Z-Up axis (Max) into Y-Up Axis (Maya)</string>
            </property>
            <property name="text">
             <string>Flip Up-Axis (Z-Y)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chk_ForceEdges">
            <property name="text">
             <string>Force Hard/Soft Edges</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chk_LockNormals">
            <property name="text">
             <string>Lock Normals</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_DeleteSelected">
       <property name="toolTip">
        <string>Delete Selected FBX(s)</string>
       </property>
       <property name="text">
        <string>Delete Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_DeleteAll">
       <property name="toolTip">
        <string>Delete All FBXs in current Working Folder</string>
       </property>
       <property name="text">
        <string>Delete All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
